app/controllers/users_controller.rb

```diff
@@ -32,6 +32,10 @@ def show
     @sales = current_user.events
   end
 
+  def index
+    @users = User.all.select{|user| user.store_admin?}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:store_admin?` as an argument to `select` instead of a block.